test/AlgebraicGeometry/Schemes/DerivedPushforward.jl

```diff
@@ -0,0 +1,51 @@
+@testset "Variation of Hodge structures on elliptic curves" begin
+  # We define a family of elliptic curves over a 1-dimensional base
+  A, (t,) = polynomial_ring(QQ, [:t])
+  P = projective_space(A, [:x, :y, :z])
+  S = homogeneous_coordinate_ring(P)
+  n = ngens(S)
+  f1 = sum(x^n for x in gens(S))
+  f0 = prod(gens(S))
+  #f0 = S[1]^n
+  # For t=0 this degenerates to three lines and we expect jumps in cohomology there
+  f = (1-t)*f0 + t*f1
```

Comment:
We compute the `H^1(\Omega^1_C)` for this family of elliptic curves varying with parameter `t`. For `t=0` this degenerates to three lines and we find jumps in the cohomology groups.

Reply:
What morphism are you pushing forward along? In the signature I just see a projective space and a module?
Or is it the structure morphism to Spec K?

Reply:
Yes, it's the structure morphism. But that's a dummy argument really, because `K` is already present as the coefficient ring of the base ring of the module. So we might want to turn this into an internal method taking only one argument. 

Keep in mind that `K` need not be a field. That's the whole point of this more general machine. In this example, it's `QQ[:t]`.